src/main/resources/allowed-labels.properties

```diff
@@ -0,0 +1,71 @@
+# This file lists all the labels that are whitelisted
+
+administrative-monitor
+analysis
+android
+aws
+bitbucket
+bitbucket-server
+builder
+buildwrapper
+ca-apm
+cli
+cloud
+cluster
+cmp
+codebuild
+credential
+database
+deployment
+deprecated
```

Comment:
```suggestion
```

Reply:
Wait, you want to disallow the deprecated label? isn't that an important one?

Reply:
It’s in the governance section